_includes/sidenav.html

```diff
@@ -15,7 +15,20 @@
 
 		<li class="dropdown-menu"><a href="{{ site.baseurl }}/63A/">SP 800-63A</a></li>
 		<li class="dropdown-menu"><a href="{{ site.baseurl }}/63B/">SP 800-63B</a></li>
+      <a href="{{ site.baseurl }}/63B/Terminology">Terminology</a>
+      <a href="{{ site.baseurl }}/63B/AAL">Authenticator Assurance Levels</a>
+      <a href="{{ site.baseurl }}/63B/Authenticators">Authenticators</a>
+      <a href="{{ site.baseurl }}/63B/Lifecycle">Lifecycle Management</a>
+      <a href="{{ site.baseurl }}/63B/Session">Session Management</a>
+      <a href="{{ site.baseurl }}/63B/SampleAuthenticators">Sample Authenticators</a>
```

Comment:
Do these new links need to be enclosed inside a
    ```<li class="dropdown-item">  </li>```
like the 63C sections are?